tests/test_skills/test_apy_estimation_abci/test_behaviours.py

```diff
@@ -1349,6 +1351,57 @@ def test_task_not_ready(
             == self.behaviour_class.behaviour_id
         )
 
+    def test_optimize_behaviour_study_case(
+        self,
+        tmp_path: PosixPath,
+        real_case_optimize_task_input_data: Tuple[
+            Dict[str, pd.DataFrame], Dict[str, Union[int, float]]
+        ],
+    ) -> None:
```

Comment:
This test proves that the behaviour completes successfully with the real case's data.